Tests/XMTPTests/ClientTests.swift

```diff
@@ -30,6 +30,20 @@ class ClientTests: XCTestCase {
 		XCTAssertFalse(cannotMessage)
 	}
 
+  func testStaticCanMessage() async throws {
+      try TestConfig.skip(because: "run manually against dev")
+    let opts = ClientOptions(api: ClientOptions.Api(env: .local, isSecure: false))
```

Comment:
It seems like this is running against local not dev?
```suggestion
      try TestConfig.skip(because: "run manually against local")
    let opts = ClientOptions(api: ClientOptions.Api(env: .local, isSecure: false))
```

Reply:
if you run the test locally you will get the following error:

`error: -[XMTPTests.ClientTests testStaticCanMessage] : failed: caught error: "creationError("setup error")"`

you should run it pointing to `dev` to pass the test